### PR TITLE
br: refine cmd stream restore to restore log

### PR DESCRIPTION
**Pull request by 3pointer**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/33795

Problem Summary:
1.  refine cmd stream restore to restore log
2. hidden not support flags for new command stream.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code
image

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by YuJuncen** [comment]
```[2022-04-07T11:13:37.443Z] level=info msg="[runner] linters took 56.437100463s with stages: goanalysis_metalinter: 56.116507985s"
[2022-04-07T11:13:37.443Z] br/pkg/storage/azblob.go:57:18: Error return value of `flags.MarkHidden` is not checked (errcheck)
[2022-04-07T11:13:37.443Z] 	flags.MarkHidden(azblobEndpointOption)
[2022-04-07T11:13:37.443Z] 	                ^
[2022-04-07T11:13:37.443Z] br/pkg/storage/azblob.go:58:18: Error return value of `flags.MarkHidden` is not checked (errcheck)
[2022-04-07T11:13:37.443Z] 	flags.MarkHidden(azblobAccessTierOption)
[2022-04-07T11:13:37.443Z] 	                ^
[2022-04-07T11:13:37.443Z] br/pkg/storage/azblob.go:59:18: Error return value of `flags.MarkHidden` is not checked (errcheck)
[2022-04-07T11:13:37.443Z] 	flags.MarkHidden(azblobAccountName)
```

**Review by YuJuncen — APPROVED** [review]
LGTM

File: br/pkg/task/restore.go [hunk 1]
```diff
@@ -132,6 +139,12 @@ type RestoreConfig struct {
 	DdlBatchSize uint `json:"ddl-batch-size" toml:"ddl-batch-size"`
 
 	WithPlacementPolicy string `json:"with-tidb-placement-mode" toml:"with-tidb-placement-mode"`
+
+	// FullBackupStorage is used to find the maps between table name and table id during restoration.
```
Comment (joccau) [inline br/pkg/task/restore.go:143]:
```suggestion
	// FullBackupStorage is used to  run `restore full` before `restore log`.
```

File: br/pkg/task/restore.go [hunk 2]
```diff
@@ -132,6 +139,12 @@ type RestoreConfig struct {
...
+	// if not specified. we cannot apply kv directly.
```
Comment (joccau) [inline br/pkg/task/restore.go:144]:
```suggestion
	// if it is empty, directly take restoring log justly.
```

**Review by joccau — COMMENTED** [review]
Rest LGTM

**Review by joccau — APPROVED** [review]

**Pull request merged**